src/bgem/gmsh/gmsh.py

```diff
@@ -645,7 +784,20 @@ def dimtagreg(self):
 
     def set_mesh_step(self, step):
```

Comment:
Rename to 'mesh_step' and return 'self', that allows usage like:

        circle = goem.circle(...).mesh_step(dx)

Reply:
done